client/packages/common/src/authentication/api/operations.graphql

```diff
@@ -148,6 +148,13 @@ query permissions($storeId: String!) {
   }
 }
 
+query preferences($storeId: String!) {
+  preferences(storeId: $storeId) {
+    showContactTracing
+    allowTrackingOfReceivedStockByDonor
+  }
+}
```

Comment:
Hi, this query shouldn't need to be here, can be removed - usePreference hook will accept the pref you are trying to query for, and build graphql query dynamically, usage like this:

https://github.com/msupply-foundation/open-msupply/blob/69e612a5b9269a704ef5a61cf23d5ed438db8e14/client/packages/system/src/Patient/PatientView/AddButton.tsx#L34